src/renderer/components/pages/HomePage.tsx

```diff
@@ -201,13 +257,61 @@ export class HomePage extends React.Component<IHomePageProps, IHomePageState> {
     this.props.clearSearch();
   }
 
+  private onFlashClick = () => {
+    this.props.onSearch('!flash');
+  }
+
+  private onShockwaveClick = () => {
+    this.props.onSearch('!shockwave');
+  }
+
+  private onGrooveClick = () => {
+    this.props.onSearch('!3d groove gx');
+  }
+
+  private on3dviaClick = () => {
+    this.props.onSearch('!3dvia');
+  }
+
+  private onActivexClick = () => {
+    this.props.onSearch('!activex');
+  }
+
+  private onHtml5Click = () => {
+    this.props.onSearch('!html5');
+  }
+
+  private onJavaClick = () => {
+    this.props.onSearch('!java');
+  }
+
+  private onPopcapClick = () => {
+    this.props.onSearch('!popcap');
+  }
+
+  private onSilverlightClick = () => {
+    this.props.onSearch('!silverlight');
+  }
+
+  private onUnityClick = () => {
+    this.props.onSearch('!unity');
+  }
+
   private getHallOfFameBrowseRoute = (): string => {
     const defaultLibrary = this.props.libraryData.libraries.find(library => !!library.default);
     const defaultRoute = defaultLibrary ? joinLibraryRoute(defaultLibrary.route) : Paths.BROWSE;
     let hof = findHallOfFamePlaylist(this.props.central.playlists.playlists);
     if (hof && hof.library) { return joinLibraryRoute(hof.library); }
     else                    { return defaultRoute;                  }
   }
+
+  private getFavoriteBrowseRoute = (): string => {
+    const defaultLibrary = this.props.libraryData.libraries.find(library => !!library.default);
+    const defaultRoute = defaultLibrary ? joinLibraryRoute(defaultLibrary.route) : Paths.BROWSE;
+    let hof = findFavoritePlaylist(this.props.central.playlists.playlists);
```

Comment:
"hof" stands for "Hall of Fame". Consider renaming it to something more fitting like "fav" or "favorites".